server/src/controllers/authentication.ts

```diff
@@ -19,4 +20,26 @@ const login = async (req: Request, res: Response) => {
   res.cookie('token', token);
   return { status: 200, msg: 'logged in successfully' };
 };
-export default login;
+const freelancerSignUp = async (req: Request, res: Response) => {
+  const {
+    Title, Major, portfolio, description, image, id,
+  } = req.body;
+  const user: UserInstance | null = await User.findOne({ where: { id } });
+  if (!user) throw serverErrs.BAD_REQUEST('user id doen\'t exist');
+  const { name, role } = user;
+  const token = await generateToken({ userID: id, role, name });
```

Comment:
First we have to `create` into freelancer model,
then we can generate token, specially that token `id` is freelancer id not `userId`